### PR TITLE
Fixed sequential plot with points

### PR DESCRIPTION
**Pull request by vandenman**
Fixes https://github.com/jasp-stats/jasp-test-release/issues/296

the plots now look like this:
screenshotFix1

In addition, I noticed there were some white borders in the bottom analysis so I made those transparent like the rest.

### CODE REVIEW

File: JASP-Engine/JASPgraphs/R/PlotRobustnessSequential.R [hunk 1]
```diff
@@ -22,6 +22,10 @@
 #' @param addLineAtOne Logical, should a black line be draw at BF = 1?
 #' @param bty List of three elements. Type specifies the box type, ldwX the width of the x-axis, lwdY the width of the y-axis.
 #' @param plotLineOrPoint String, should the main geom in the plot be a line or a point?
+#' @param pointShapesSmallN String, if \code{plotLineOrPoint == "point"} then this controls the shape aestethic.
+#' @param pointFillsSmallN String, if \code{plotLineOrPoint == "point"} then this controls the fill aestethic.
```
Comment (TimKDJ) [inline JASP-Engine/JASPgraphs/R/PlotRobustnessSequential.R]: aesthetic ;)

File: JASP-Engine/JASPgraphs/R/PlotRobustnessSequential.R [hunk 2]
```diff
@@ -22,6 +22,10 @@
 #' @param addLineAtOne Logical, should a black line be draw at BF = 1?
 #' @param bty List of three elements. Type specifies the box type, ldwX the width of the x-axis, lwdY the width of the y-axis.
 #' @param plotLineOrPoint String, should the main geom in the plot be a line or a point?
+#' @param pointShapesSmallN String, if \code{plotLineOrPoint == "point"} then this controls the shape aestethic.
```
Comment (TimKDJ) [inline JASP-Engine/JASPgraphs/R/PlotRobustnessSequential.R]: .. and for large N?

Comment (vandenman) [reply JASP-Engine/JASPgraphs/R/PlotRobustnessSequential.R]: for large N (> 60) people should use a line (e.g., `plotLineOrPoint = "line"`). I didn't invent that cut-off that's simply what was used before. At that point, the code automagically switches to using line types.

File: JASP-Engine/JASPgraphs/DESCRIPTION [hunk 3]
```diff
@@ -1,7 +1,7 @@
 Package: JASPgraphs
 Type: Package
 Title: Custom graphs for JASP
-Version: 0.3.5
+Version: 0.3.6
```
[the remark this answers is not in the document]
Comment (vandenman) [reply JASP-Engine/JASPgraphs/DESCRIPTION]: hmm, I thought I did run `manageTestPlots()` and not a single plot changed, but apparently, the file didn't update. I'll do it again!

File: JASP-Engine/JASPgraphs/R/PlotRobustnessSequential.R (hunk 2 above)
Comment (TimKDJ) [reply JASP-Engine/JASPgraphs/R/PlotRobustnessSequential.R]: Then `pointShape` is an okay name, right? Considering you never get points for > smallN you don't need to specify that

**Comment by TimKDJ** [comment]
This looks suboptimal:

Screenshot 2019-09-03 at 12 59 35

**Comment by vandenman** [comment]
Okay, so now the points plot looks like this:

afbeelding

and the lines plot like this:

afbeelding

In the last commit, I also removed an unused function (Ctrl+shift+F did not yield any results other than the definition).

Let me know what you think!

**Pull request merged**